Dockerfile

```diff
@@ -138,8 +138,8 @@ ENV PATH=/opt/conda/bin:/root/.local/bin/:$PATH
 RUN cat /tmp/requirements/pipx.txt | xargs -rn 1 pipx install && \
     pipx list --json && \
     # This is TMP work-around due to https://github.com/neuro-inc/neuro-cli/pull/2671
-    pipx runpip neuro-all uninstall -y click && \
-    pipx runpip neuro-all install click==8.1.3 && \
+    pipx runpip apolo-all uninstall -y click && \
+    pipx runpip apolo-all install click==8.1.3 && \
```

Comment:
let's drop it since the issue is resolved

Reply:
removed